.eslintrc.js

```diff
@@ -166,7 +166,13 @@ module.exports = {
     'no-undef-init': 0,
     'no-undef': 1,
     'no-undefined': 0,
-    'no-unused-vars': [1, { args: 'none' }],
+
+    'no-unused-vars': 'off',
+    '@typescript-eslint/no-unused-vars': [
+      'error',
+      { argsIgnorePattern: '^_', ignoreRestSiblings: true },
```

Comment:
I prefer `argsIgnorePattern: '^_'` over `args: 'none'` to explicitly add `_` suffix to arguments that should be keeped